src/test/rustdoc/issue-74134-4.rs

```diff
@@ -0,0 +1,11 @@
+// compile-flags: --document-private-items
```

Comment:
Ditto, this can be one test case. See https://github.com/rust-lang/rust/pull/72771/files#diff-5aeb32b32ed1ac2b564f4f43c17bbdeeR3 for an example.

Reply:
Done. See my other reply on issue-74134-2.